lib/routes/account.js

```diff
@@ -1646,11 +1668,46 @@ module.exports = function (
             isVerified = isVerified && sessionVerified
           }
 
-          return {
-            email: sessionToken.email,
-            verified: isVerified,
-            sessionVerified: sessionVerified,
-            emailVerified: emailVerified
+          if (! isVerified) {
+            return bounces.check(sessionToken.email)
```

Comment:
Would it be worth adding in a 'feature' switch, allowing us to try this at like 10% users and see if the impact would be too much?